website/docs/getting-started/setup/upgrade-to-business-edition/README.md

```diff
@@ -20,6 +20,12 @@ Upgrade your Community Edition to a Business Edition by selecting below installa
             <b><a href="/getting-started/setup/upgrade-to-business-edition/docker">Docker</a></b>
         </div>
         <div class="columnGrid column-two" align="center">
+            <div class="containerCol">
```

Comment:
logo appears to be broken
<img width="870" alt="Screenshot 2022-12-19 at 17 40 26" src="https://user-images.githubusercontent.com/32227135/208423338-d8282a72-47c3-4b4f-aa85-e722a3bfe7ef.png">